pkg/metrics/instance/instance.go

```diff
@@ -843,7 +843,10 @@ func newScrapeManager(logger log.Logger, app storage.Appendable) *scrape.Manager
 	// data race of modifying that global, we lock a mutex here briefly.
 	managerMtx.Lock()
 	defer managerMtx.Unlock()
-	return scrape.NewManager(logger, app)
+	options := &scrape.Options{
+		ExtraMetrics: false,
```

Comment:
What are the extra metrics here? Does Prometheus default to true?

Reply:
this defaults to `false`, but looking at the code in prometheus, we should be able to just pass nil here:
https://github.com/prometheus/prometheus/blob/main/scrape/manager.go#L103-L105

Reply:
https://prometheus.io/docs/prometheus/latest/feature_flags/#extra-scrape-metrics

Reply:
@rfratto resolving this, doesn't seem like something we need/want to support in agent. Let me know if you disagree and we can unresolve.

Reply:
Let's open an issue for this to make this configurable. It's an experimental feature in Prometheus, so we might not want to do it right away, but we should have an issue to start a discussion around it.